processing/src/main/java/io/druid/query/DruidMetrics.java

```diff
@@ -59,6 +61,7 @@ public static int findNumComplexAggs(List<AggregatorFactory> aggs)
   public static <T> ServiceMetricEvent.Builder makePartialQueryTimeMetric(Query<T> query)
   {
     return new ServiceMetricEvent.Builder()
+        .setDimension(VERSION, Strings.nullToEmpty(DruidMetrics.class.getPackage().getImplementationVersion()))
```

Comment:
it's probably better to leave it null than have it empty string, metrics should support null dimension values